File: 51주차/튜플.js
function solution(s) {
  var answer = [];
  let set = new Set();

  s = s.slice(2, s.length - 2);
  s = s.split("},{");
  s = s.map((v) => v.split(",").map((e) => +e));

  s.sort((a, b) => a.length - b.length);
  s.forEach((arr) => {
    arr.forEach((v) => set.add(v));
  });

  return [...set];
}
// 프로그래머스.

File: 67주차/튜플.py
def solution(s):
    answer =[]
    s = s[2:-2]
    s = s.split("},{")
    s.sort(key = lambda x: len(x))
    tmp = []
    check = set()
    for arr in s:
        tmp.append(list(map(int,arr.split(","))))
    for arr in tmp:
        for v in arr:
            if v not in check:
                check.add(v)
                answer.append(v)
            
    return answer